src/modules/back-to-top/back-to-top.component.ts

```diff
@@ -0,0 +1,35 @@
+import { Component, ElementRef, Input, OnInit, ViewChild } from '@angular/core';
+
+@Component({
+  selector: 'stache-back-to-top',
+  templateUrl: './back-to-top.component.html',
+  styleUrls: ['./back-to-top.component.scss']
+})
+export class StacheBackToTopComponent {
+  @Input() offset: number;
```

Comment:
(There is a tslint config option for this somewhere, and I know SKY UX uses it on Builder. Let me attempt to apply that to our code base as well.)